test/connectivity/scripts/connectivityTest.sh

```diff
@@ -334,6 +339,19 @@ function run_connectivity_test() {
     print_highlighted_message "Run connectivity test with -d '$device_id' started at $test_start_time"
 
     SECONDS=0
+
+    case $image_architecture_label in
```

Comment:
optimize_for_performance variable already exists in line 554; please move/merge this block to line 552.  Also please use lower case for internal variable (since it is not passing in).